src/styles/blocks/card.scss

```
@@ -0,0 +1,115 @@
+.card {
+  box-sizing: border-box;
+  width: $card-size;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-self: center;
+  padding: 32px var(--elements-position) 16px var(--elements-position);
+  border: 1px solid #e5e5e5;
+  transition: all 300ms linear;
```

```suggestion
  transition: all 300ms linear;
```

avoid using `all`,  specify the properties